### ISSUE TITLE
MagicClothing_Inpainting failed 

### ISSUE THREAD

**Issue by FourTest** [frankchieng/ComfyUI_MagicClothing#54]
Error occurred when executing MagicClothing_Inpainting:
Sizes of tensors must match except in dimension 1. Expected size 3 but got size 2 for tensor number 1 in the list.

error trace:

File "/data/users/ComfyUI/execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "/data/users/ComfyUI/execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "/data/users/ComfyUI/execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "/data/users/ComfyUI/custom_nodes/ComfyUI_MagicClothing/nodes.py", line 107, in cloth_inpainting
images, cloth_mask_image = full_net.generate(cloth_image, cloth_mask_image, kwargs['prompt'], a_prompt, kwargs['num_samples'], kwargs['negative_prompt'], kwargs['seed'], kwargs['guidance_scale'], kwargs['cloth_guidance_scale'], kwargs['sample_steps'], kwargs['height'], kwargs['width'], image=person_image,mask_image=person_mask,control_image=control_img)
File "/data/users/ComfyUI/custom_nodes/ComfyUI_MagicClothing/garment_adapter/garment_diffusion.py", line 109, in generate
images = self.pipe(
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
File "/data/users/ComfyUI/custom_nodes/ComfyUI_MagicClothing/pipelines/VirtualTryOnPipeline.py", line 429, in __call__
noise_pred = self.unet(
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
return forward_call(*args, **kwargs)
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/diffusers/models/unets/unet_2d_condition.py", line 1121, in forward
sample, res_samples = downsample_block(
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
return forward_call(*args, **kwargs)
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/diffusers/models/unets/unet_2d_blocks.py", line 1199, in forward
hidden_states = attn(
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
return forward_call(*args, **kwargs)
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/diffusers/models/transformers/transformer_2d.py", line 391, in forward
hidden_states = block(
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
return forward_call(*args, **kwargs)
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/diffusers/models/attention.py", line 335, in forward
attn_output = self.attn1(
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
return forward_call(*args, **kwargs)
File "/data/users/py3.10_for_ComfyUI/lib/python3.10/site-packages/diffusers/models/attention_processor.py", line 512, in forward
return self.processor(
File "/data/users/ComfyUI/custom_nodes/ComfyUI_MagicClothing/garment_adapter/attention_processor.py", line 294, in __call__
hidden_states = torch.cat([hidden_states, ref_hidden_states], dim=1)
